locations/spiders/shell.py

```diff
@@ -12,14 +22,28 @@ def parse_item(self, item, location):
         amenities = location["amenities"]
         fuels = location["fuels"]
         apply_category(Categories.FUEL_STATION, item)
-        if "shop" in amenities:
+        if "twenty_four_hour" in amenities:
+            item["opening_hours"] = OpeningHours()
+            item["opening_hours"].add_days_range(DAYS, "00:00", "23:59")
+        if "shop" in amenities or "selectshop" in amenities:
             apply_category(Categories.SHOP_CONVENIENCE, item)
-        apply_yes_no("amenity:chargingstation", item, "electric_charging_other" in fuels or "shell_recharge" in fuels)
-        apply_yes_no("toilets", item, "toilet" in amenities)
-        apply_yes_no("atm", item, "atm" in amenities)
-        apply_yes_no("car_wash", item, "carwash" in amenities)
-        apply_yes_no("hgv", item, "hgv_lane" in amenities)
-        apply_yes_no("wheelchair", item, "disabled_facilities" in amenities)
+        if "restaurant" in amenities:
+            apply_category(Categories.RESTAURANT, item)
+        if "charging" in amenities or "electric_charging_other" in fuels or "shell_recharge" in fuels:
+            apply_category(Categories.CHARGING_STATION, item)
```

Comment:
`shell_recharge` is big enough to be it's own POI (and has it's own spider). I don't know the about the others.

If we want to collect them lets use `fuel:electric=yes`, it's not used in OSM, but `amenity=charging_station;fuel` breaks compatibility with OSM.

Reply:
`fuel:electricity` is a more common tag (1.6k vs 12 uses for `fuel:electric`) but it's used for BBQs. Maybe we can create a discussion on the forum and see what other mappers think?

Reply:
Should've checked Overpass before writing the previous reply. Of those 1.6k elements with `fuel:electricity`, ~1.3k are on `amenity=fuel`. I think that it should do the job.

Reply:
I'm good with `fuel:electricity`. I don't do the tagging mailing list, but you are welcome to reach out.

```suggestion
        apply_yes_no("fuel:electricity", item, True)
```